docs/src/main/asciidoc/tests-with-coverage.adoc

```diff
@@ -175,6 +175,9 @@ Now we need to add Jacoco to our project. To do this we need to add the followin
 This Quarkus extension takes care of everything that would usually be done via the Jacoco maven plugin, so no additional
 config is required.
 
+WARNING: Do not use both the extension and the plugin, if you configure both you will get lots of errors about classes
+already being instrumented.
```

Comment:
Doesn't that contradict paragraph "7. Coverage for tests not using @QuarkusTest"?

Reply:
@stuartwdouglas could you have a look at this comment?